platform/website/src/routes/+page.svelte

```diff
@@ -463,7 +462,8 @@
 
 		& > .title {
 			margin-left: 1rem;
-			margin-bottom: -1.5rem;
+			/* -2rem because categories has padding-top of 2rem */
+			margin-bottom: calc(-2rem + 1rem);
```

Comment:
Why is this calculated? is it not just -1rem since the units are the same?

Reply:
I though it might be easier to understand like this, only for readability

Reply:
I think this gets rendered to front end, rather add a comment saying why -1rem and show -2rem + 1 rem = -1 or whatever

Reply:
Sass will precalculate it: https://sass-lang.com/documentation/values/calculations/

I can still change it if you want

Reply:
I think, yeah rather change to be a comment, unless you want to switch to using variables rather than fixed values.